GeorgianMemenator/Scenes/DogBreedDetail/DogBreedDetailViewModel.swift

```diff
@@ -7,8 +7,6 @@
 
 import Foundation
 
-
```

Comment:
Please be careful what for these unneeded empty lines, its fine for now, but it would be nice if you remove them in future.

Reply:
@qetiavanashvili yes, now its about removing. You removed empty lines, but try to make only changes that makes sense. Your purpose in this pull request is not removing lines. Additional line changes makes pull request review process slower and difficult.

But its fine for now, you can merge, something to keep in mind in future 🚀

Reply:
okey, I see.